lib/package-managers/npm.js

```diff
@@ -10,16 +11,49 @@ const versionUtil = require('../version-util')
 
 const TIME_FIELDS = ['modified', 'created']
 
+const npmConfigToPacoteMap = {
+  cafile: path => {
+    // load-cafile, based on github.com/npm/cli/blob/40c1b0f/lib/config/load-cafile.js
+    if (!path) return
+    const cadata = fs.readFileSync(path, 'utf8')
+    const delim = '-----END CERTIFICATE-----'
+    const output = cadata
+      .split(delim)
+      .filter(xs => !!xs.trim())
+      .map(xs => `${xs.trimLeft()}${delim}`)
+    return { ca: output }
+  },
+  'fetch-retries': 'fetchRetries',
+  'fetch-retry-factor': 'fetchRetryFactor',
+  'fetch-retry-mintimeout': 'fetchRetryMintimeout',
+  'local-address': 'localAddress',
+  'prefer-offline': 'preferOffline',
+  'prefer-online': 'preferOnline',
+  'strict-ssl': 'strictSSL',
+  'user-agent': 'userAgent',
```

Comment:
These appear to be a regular transformation to PascalCase. Can you use https://github.com/sindresorhus/camelcase to generalize this? Are there any false positives?

`strict-ssl` might be an exception since `SSL` is capitalized, but an exception for a single key is better than making every key explicit.

Reply:
You have lodash in this file, and it has a `camelCase` method. I was thinking that using a precomputed value was more straight forward than having `_.camelCase` listed a bunch of times. And, yes, strict-ssl would be an exception.

Reply:
Yes, we could use lodash.

The idea would be not to have multiple places where `_.camelCase` is used, but to abstract it so that `_.camelCase` only appears once in the code, inside [libnpmconfig.read().forEach](https://github.com/raineorshine/npm-check-updates/pull/713/files#diff-036fe149039aa46377f04ef192bfe426R39) where it will automatically be called on every key. Are there any false positives that would prevent us from doing that?

Reply:
No false positives that I can see, but there is a false negative (maxsockets -> maxSockets) that I missed in the list before. I will adjust the PR.

Reply:
Bah, I just remembered that there are extra values that `libnpmconfig` adds to the returned config. I will need to make sure those are not affected by simply camel casing all special case keys.